src/frontend/components/UI/Header/index.css

```diff
@@ -14,7 +14,7 @@
 
 .frameless:not(.fullscreen) .Header {
   padding-right: calc(
-    var(--overlay-controls-width) - env(titlebar-area-x, 0px)
+    var(--overlay-controls-width) - env(titlebar-area-x, 0px) + 10px
```

Comment:
the frameless window controls were overlapping the Filters button

![image](https://github.com/user-attachments/assets/ef10de2d-82a3-4bcd-9dc0-623d682150b3)

and now

![image](https://github.com/user-attachments/assets/c37da711-c441-4e25-afa1-6e7f45769b49)